.github/workflows/pull_request.yml

```diff
@@ -24,6 +24,14 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
+      - name: Install System Dependencies
```

Comment:
I believe the correct syntax here would be to name each condition separately:

```yaml
    - name: condition no 1
      if: condition_1=true
      run: script for condition 1
   - name: condition 2
      if: condition_2=true
      run: script for condition 2
```

Reply:
Okay, that makes sense. I have pushed the changes, sorry for the inconvenience.

Reply:
The test failed since I used double quotes instead of using single quotes for specifying the system.

Reply:
Thanks for all your efforts. Looks good to me so far 😊

Are you able to see the logs of the Github actions? There seem to be some linting issues. If you have no access I can paste the logs here ☺️

Reply:
Yup I have modified my code according to the error. :)